docs/index.md

```diff
@@ -1,227 +1,151 @@
+        <div class="col-8 p-stepped-list__content" >
+MicroK8s will install a minimal, lightweight Kubernetes you can run and use on practically any machine. It can be installed with a snap:
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="sudo snap install microk8s --classic --channel=1.16/stable" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <script id="asciicast-279765" src="https://asciinema.org/a/279765.js" async data-autoplay="true" data-rows="4"></script>
+          <p>
+          <a href="/docs/setting-snap-channel">More about setting the channel&nbsp;›</a>
+          </p>
+        </div>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title"><span class="p-stepped-list__bullet">2</span>Check the status</h3>
+        <div class="p-stepped-list__content">
+
+MicroK8s has a built-in command to display its status. During installation you
+can use the `--wait-ready` flag to wait for the Kubernetes services to initialise:
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="microk8s.status --wait-ready" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+        </div>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title"><span class="p-stepped-list__bullet">3</span>Access Kubernetes</h3>
+        <div class="p-stepped-list__content">
+<p>MicroK8s bundles its own version of <code>kubectl</code> for accessing Kubernetes. Use it to run
+commands to monitor and control your Kubernetes. For example, to view your node:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="microk8s.kubectl get nodes" readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+<p>.. or to see the running services: </p>
+<div class="p-code-copyable">
+  <input class="p-code-copyable__input" value="microk8s.kubectl get services" readonly="readonly">
+  <button class="p-code-copyable__action">Copy to clipboard</button>
+</div>
+        </div>
+
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title"><span class="p-stepped-list__bullet">4</span>Deploy an app</h3>
+        <div class="p-stepped-list__content">
+          <p>Of course, Kubernetes is meant for deploying apps and services. You can use
+          the <code>kubectl</code> command to do that as with any Kuberenetes. Try
+          installing a demo app:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="microk8s.kubectl create deployment kubernetes-bootcamp --image=gcr.io/google-samples/kubernetes-bootcamp:v1" readonly="readonly">
```

Comment:
@ktsakalozos You may have a better idea for a demo?

Reply:
There is --image=dontrebootme/microbot:v1 if you prefer.

Reply:
I'm ambivalent, it's your show!